docker-compose.yml

```diff
@@ -92,8 +92,22 @@ services:
     depends_on:
       - minio
 
+  dms:
+    image: adfinissygroup/document-merge-service:v4.4.1
+    depends_on:
+      - db
+    volumes:
+      - templatefiles:/var/lib/document-merge-service/media
+    environment:
+      - DATABASE_ENGINE=django.db.backends.postgresql
+      - DATABASE_HOST=db
+      - DATABASE_PORT=5432
+      - DATABASE_NAME=dms
+      - DATABASE_USER=dms
```

Comment:
Shouldn't the database user be put in the `.override.yml` file instead? That way it won't be forgotten when deploying prod

Reply:
As of now, all db users are configured in `docker-compose.yml`, if they differ from the default.